configargparse.py

```diff
@@ -946,10 +947,14 @@ def format_help(self):
             msg += (" If an arg is specified in more than one place, then "
                 "commandline values override %s.") % (
                 " which override ".join(value_sources))
-        if msg:
-            self.description = (self.description or "") + " " + msg
 
-        return argparse.ArgumentParser.format_help(self)
+        text_width = max(self._get_formatter()._width, 11)
+        return textwrap.fill(msg, text_width)
```

Comment:
I don't think text-wrapping should be handled here. If anywhere, it should be in the HelpFormatter.
Would prefer minimal changes necessary to move the config file docs to the end of the help message.

Reply:
These are minimal changes, esp. as I would rather not spend time understanding and extending the relevant HelpFormatter :).